src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -599,9 +597,12 @@ void AutomatedRssDownloader::updateFieldsToolTips(bool regex)
               + "<li>" + tr("Whitespaces count as AND operators (all words, any order)") + "</li>"
               + "<li>" + tr("| is used as OR operator") + "</li></ul></p>"
               + "<p>" + tr("If word order is important use * instead of whitespace.") + "</p>";
-        ui->lineContains->setToolTip(tip);
-        ui->lineNotContains->setToolTip(tip);
     }
+
+    // Whether regex or wildcard, warn about a potential gotcha for users.
+    tip += "<p>" + tr("An expression with an empty %1 clause (e.g. %2)").arg("<tt>|</tt>").arg("<tt>expr|</tt>");
```

Comment:
[tr()](https://doc.qt.io/qt-5/qobject.html#tr) can take up to 3 parameters. I was talking about the 2nd parameter, which disambugates the sourceText for the translations. It is, let's say, instructions/explanations for them.
That why I proposed:
```c++
"<p>" + tr("An expression with an empty %1 clause (e.g. %2)", "We talk about regex/wildcards in the RSS filters section here. So a valid sentence would be: An expression with an empty | clause (e.g. expr|)").arg("<tt>|</tt>").arg("<tt>expr|</tt>");
```